scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -35,6 +35,8 @@
   $(document).ready(function () {
       var div = $("#igvDiv")[0],
               options = {
+                    loadDefaultGenomes: false,
```

Comment:
👍🏻 I'm also looking at this suggested solution: https://github.com/igvteam/igv.js/issues/1340#issuecomment-854838850